README.md

```diff
@@ -131,6 +131,17 @@ The current version of the website is served out of the `master` branch.
 
 All versions of the site that relate to past and future versions will be named after their Kubernetes release number. For example, [the old branch for the 1.1 docs is called `release-1.1`](https://github.com/kubernetes/kubernetes.github.io/tree/release-1.1).
 
+## Partners {#partners}
```

Comment:
All headings automatically get an anchortag, at least in our Markdown processor. :)